src/js/contexts/AnnounceContext/doc.js

```diff
@@ -13,7 +13,11 @@ export const doc = AnnounceContext => {
   DocumentedAnnounceContext.propTypes = {
     children: PropTypes.func.description(
       `Render function that will be called with an 'announce' function that
-      can be called when something should be announced.`,
+      can be called when something should be announced. 
+      'announce' function accepts 'message', 'mode' and 'timeout' as arguments
+      and these arguments can be passed as 'props' to the return component.
+      Example:  {announce => <Announcer announce={announce} {...props} />}
```

Comment:
I think having `Announcer` here makes it look like a grommet component. How about something like:
```
{announce => <Button onClick={() => announce("Button was clicked", "polite", 1000)}
```
I think we should also mention possible values for `mode` and the units for `timeout`.

Reply:
Updated docs.